data/multus/001-rbac.yaml

```diff
@@ -36,3 +36,19 @@ kind: ServiceAccount
 metadata:
   name: multus
   namespace: multus
+{{ if .EnableSCC }}
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: multus
+allowPrivilegedContainer: true
+allowHostDirVolumePlugin: true
+allowHostNetwork: true
```

Comment:
why do we need host-network for multus? AFAIU the container merely copies a binary to host but plugin is executed outside the container.